test/interactive/interactive.test.js

```diff
@@ -31,9 +32,30 @@ describe('--interactive flag', () => {
                 return;
             }
             if (semaphore === 0) {
-                expect(data).toMatchSnapshot();
                 expect(data).toContain('main.js');
                 proc.kill();
+                writeFileSync(resolve(__dirname, 'index.js'), `console.log('I am Iron Man');`);
+                done();
+                return;
+            }
+        });
+    });
+
+    it('should not output in interactive with --watch only', (done) => {
+        const proc = runAndGetWatchProc(__dirname, ['-c', './webpack.watch.js', '--watch'], false, '', true);
+        let semaphore = 1;
+        const clear = '\x1B[2J\x1B[3J\x1B[H';
+        proc.stdout.on('data', (chunk) => {
+            const data = chunk.toString();
+            if (data.includes('watching files for updates')) {
+                writeFileSync(resolve(__dirname, 'index.js'), `console.log('I am Batman');`);
+                semaphore--;
+                return;
+            }
+            if (semaphore === 0) {
+                expect(data).not.toBe(clear);
+                proc.kill();
+                writeFileSync(resolve(__dirname, 'index.js'), `console.log('I am Iron Man');`);
```

Comment:
Let's use snapshots too, now we should have multiple outputs, use `.replace()` to remove time and hashes

Reply:
Done regressive phrase testing on stdout instead.